Terms contains (#1942)
Codecademy/docs: content/c-sharp/concepts/strings/terms/contains/contains.md

---
Title: '.Contains()'

> Description: 'Defines whether a certain string *contains* a substring, passed as parameter.Returns a boolean data type.'

Reviewer: ```suggestion
Description: 'Determines whether a character or subset of characters is present within a given string'
```

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Strings'
  - 'Characters'
  - 'Methods'
CatalogContent:
  - 'learn-c-sharp'
  - 'paths/computer-science'
---

> The **`.Contains()`** method defines whether the `string` includes a particular character or substring. Returns true if the character is included, else the method returns false. There is an opportunity to add extra parameters to correspond to the comparison rules.

Reviewer: ```suggestion
The **`.Contains()`** method determines whether a `string` includes a particular character or substring. It returns `true` if the character is included, otherwise the method returns `false`. There are additional parameters that can modify the comparison rules.
```

## Syntax

```pseudo

> // Defines whether the String includes a given character

Reviewer: please replace "defines" with a "determines" below

String.Contains(char);

// Defines whether the String includes a given string
String.Contains(string);

// Defines whether the String includes a given character considering the type of comparison
String.Contains(char, comparisonType)

// Defines whether the String includes a given string considering the type of comparison
String.Contains(string, comparisonType)
```

> `.Contains()` takes arguments such as:

Reviewer: ```suggestion
`.Contains()` takes the following arguments:
```

> - `char`is a single letter.

Reviewer: ```suggestion
- `char` is a single character.
```

> - `string`is a subsequence of characters.

Reviewer: ```suggestion
- `string` is a sequence of characters.
```

> - `comparisonType`is an enumeration value that sets the comparison rules.

Reviewer: ```suggestion
- `comparisonType` is an enumeration value that sets the comparison rules.
```

## Example

> 

Reviewer: Please update to include an example and the relevant output

---
Title: '.Contains()'
Description: 'Determines whether a character or subset of characters is present within a given string'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Strings'
  - 'Characters'
  - 'Methods'
CatalogContent:
  - 'learn-c-sharp'
  - 'paths/computer-science'
---

The **`.Contains()`** method determines whether a `string` includes a particular character or substring. It returns `true` if the character is included, otherwise the method returns `false`. There are additional parameters that can modify the comparison rules.

## Syntax

```pseudo
// Determines whether the String includes a given character
String.Contains(char);

// Determines whether the String includes a given string
String.Contains(string);

// Determines whether the String includes a given character considering the type of comparison
String.Contains(char, comparisonType)

// Determines whether the String includes a given string considering the type of comparison
String.Contains(string, comparisonType)
```

`.Contains()` takes the following arguments:

- `char` is a single character.
- `string` is a sequence of characters.

> - `comparisonType` is an enumeration value that sets the comparison rules.

Reviewer 2: There should be some description of the comparisonType options, if not a full blown example demonstrating them in use.
